src/differenceInBusinessDays/test.ts

```diff
@@ -65,37 +65,241 @@ describe('differenceInBusinessDays', function () {
-    const result = differenceInBusinessDays(
-      new Date(2022, 0 /* Jan */, 17),
-      new Date(2022, 0 /* Jan */, 7),
-      {
-        businessDays: [0, 1, 2, 3, 4, 5, 6],
-      }
-    )
-    assert(result === 10)
+  describe('exceptions option', function () {
+    it('can add true exceptions to include days as businessDays', function () {
+      // with businessDays as Mon-Fri
+      const result = differenceInBusinessDays(
+        new Date(2022, 0 /* Jan */, 17),
+        new Date(2022, 0 /* Jan */, 7),
+        {
+          // Adding a Saturday and Sunday as business days
+          exceptions: { '01/08/22': true, '01/09/22': true },
+        }
+      )
+
+      assert(result === 8)
+    })
+
+    it('can add false exceptions to remove days as businessDays', function () {
+      // with businessDays as Mon-Fri
+      const result = differenceInBusinessDays(
+        new Date(2022, 0 /* Jan */, 17),
+        new Date(2022, 0 /* Jan */, 7),
+        {
+          // Removing a Tues and Wed as business days
+          exceptions: { '01/11/22': false, '01/12/22': false },
+        }
+      )
+
+      assert(result === 4)
+    })
+
+    it('can handle true and false exceptions', function () {
+      // with businessDays as Mon-Fri
+      const result = differenceInBusinessDays(
+        new Date(2022, 0 /* Jan */, 17),
+        new Date(2022, 0 /* Jan */, 7),
+        {
+          // Adds a Sat and Sun, removes a Tues and Wed as business days
+          exceptions: {
+            '01/08/22': true,
+            '01/09/22': true,
+            '01/11/22': false,
+            '01/12/22': false,
+          },
+        }
+      )
+
+      assert(result === 6)
+    })
+
+    it('should only add true exceptions that are not already businessDays', function () {
+      // with businessDays as Mon-Fri
+      const result = differenceInBusinessDays(
+        new Date(2022, 0 /* Jan */, 17),
+        new Date(2022, 0 /* Jan */, 7),
+        {
+          // Adding a Sat and Sun as business days, but not Mon and Tues which are already business days
+          exceptions: {
+            '01/08/22': true,
+            '01/09/22': true,
+            '01/10/22': true,
+            '01/11/22': true,
+          },
+        }
+      )
+
+      assert(result === 8)
+    })
+
+    it('should only remove false exceptions that are businessDays', function () {
+      // with businessDays as Mon-Fri
+      const result = differenceInBusinessDays(
+        new Date(2022, 0 /* Jan */, 17),
+        new Date(2022, 0 /* Jan */, 7),
+        {
+          // Removing a Tues and Wed as business days, but not Sat and Sun which are not business days
+          exceptions: {
+            '01/11/22': false,
+            '01/12/22': false,
+            '01/15/22': false,
+            '01/16/22': false,
+          },
+        }
+      )
+
+      assert(result === 4)
+    })
+
+    it('can handle exceptions that fall on both of the argument dates', function () {
```

Comment:
This test fails. I put the result as what I _think_ it should be, so I think if both days are exceptions then one extra day should be added, but I do not have that implemented in the logic.